tests/component/task/test_triggers.py

```diff
@@ -298,3 +416,64 @@ def test___reference_trigger___cfg_anlg_multi_edge_ref_trig___no_errors(
         sim_9775_ai_voltage_multi_edge_task.triggers.reference_trigger.anlg_multi_edge_lvls
         == pytest.approx(trig_levels, abs=0.001)
     )
+
+
+@pytest.mark.parametrize(
+    "trig_src, pretrig_samples, trig_edge",
+    [
+        ("APFI0", 10, Edge.RISING),
+        ("APFI1", 20, Edge.FALLING),
+    ],
+)
+def test___reference_trigger___cfg_dig_edge_ref_trig___no_errors(
+    sim_6363_ai_voltage_task: Task,
+    trig_src: str,
+    pretrig_samples: int,
+    trig_edge: Edge,
+):
+    device_name = sim_6363_ai_voltage_task.devices[0].name
+
+    sim_6363_ai_voltage_task.timing.cfg_samp_clk_timing(1000)
+    sim_6363_ai_voltage_task.triggers.reference_trigger.cfg_dig_edge_ref_trig(
+        trigger_source=trig_src,
+        pretrigger_samples=pretrig_samples,
+        trigger_edge=trig_edge,
+    )
+
+    assert (
+        sim_6363_ai_voltage_task.triggers.reference_trigger.dig_edge_src
+        == f"/{device_name}/{trig_src}"
+    )
+    assert sim_6363_ai_voltage_task.triggers.reference_trigger.pretrig_samples == pretrig_samples
+    assert sim_6363_ai_voltage_task.triggers.reference_trigger.dig_edge_edge == trig_edge
+
+
+@pytest.mark.parametrize(
+    "trig_src, trig_pattern, pretrig_samples, trig_when",
+    [
+        ("port1/line2, port1/line4", "XX", 10, DigitalPatternCondition.PATTERN_MATCHES),
```

Comment:
All X's means "ignore all these channels". Mix it up a bit with some 0's, 1's, R's, E's, or F's. I don't care which ones you test as long as they're not all X's.

https://www.ni.com/docs/en-US/bundle/ni-daqmx/page/digpattern.html

Same for the start trigger test above.